pcdsdevices/epics_motor.py

```diff
@@ -64,31 +66,25 @@ class EpicsMotorInterface(FltMvInterface, EpicsMotor):
 
     @property
     def low_limit(self):
-        """
-        The lower soft limit for the motor.
-        """
-        return self.low_limit_travel.value
+        """The lower soft limit for the motor."""
+        return self.low_soft_limit.value
```

Comment:
These should not have been renamed, the correct names were already in use: https://github.com/bluesky/ophyd/blob/eee8797e79448a1dc02c65952af475563f4d260a/ophyd/epics_motor.py#L65-L68

Reply:
Oh that's weird, I don't remember changing those. Could that have been a result of a weird thing in my rebase?

Reply:
Probably a rebase artifact